modules/ROOT/nav.adoc

```diff
@@ -1,18 +1,23 @@
 .xref:index.adoc[API Community Manager]
-* xref:index.adoc[API Community Manager]
- ** xref:security.adoc[Security]
-* xref:admin-interface.adoc[Administrator Interface]
+* xref:index.adoc[Overview]
 * xref:setup.adoc[Set Up Your API Community]
- ** xref:before-installation.adoc[Before Installation]
+ ** xref:before-installation.adoc[Installation Pre-requisites]
  ** xref:installation.adoc[Installation]
- ** xref:branding-and-customization.adoc[Branding and Customization]
- ** xref:preview-community.adoc[Preview Community]
+* xref:brand-intro.adoc[Brand and Customize your API Community]
+ ** xref:branding-and-customization.adoc[Customize the look&feel of your API Community]
```

Comment:
For a shorter title than "Customize the look&feel of your API Community", how about "Customize community look and feel"?